main-script.cpp

```diff
@@ -0,0 +1,79 @@
+#include <iostream>
+#include "matrix-matrix-product.hpp"
+#include "matrix-vector-product.hpp"
+#include "read-matrix-market.hpp"
+#include "write-matrix-market.hpp"
+
+using namespace std;
```

Comment:
Although almost every tutorial on the internet says to do this, I would highly recommend against it. In a header file this will blow up other peoples code if they include your header. In the cpp file, it is often ok, but can still cause issues.